packages/amplify-dotnet-function-template-provider/resources/lambda/Crud/event.json.ejs

```diff
@@ -1,7 +1,7 @@
 {
   "body": null,
   "resource": "<%= props.functionTemplate.parameters.path %>/{proxy+}",
-  "path": "<%= props.functionTemplate.parameters.path %>/",
+  "path": "<%= props.functionTemplate.parameters.path %>/<%= props.functionTemplate.parameters.database.tableDefinition.partitionKeyName %>:foo",
```

Comment:
while we don't use this event in test I was playing a bit with this in aws console to invoke resulting lambda.
previous version was leading to error response asking for missing path parameter - this change fixes this.